src/components/security/security-controller.ts

```diff
@@ -24,8 +24,8 @@ export async function securityGet(req: Request, res: Response): Promise<void> {
       ? req.session.mfaMethods.map((method) => {
```

Comment:
Perhaps it is worth renaming `method` to `mfaMethod` so the call is mfaMethod.method rather than method.method which is slightly confusing or just deconstruct it at top level so the calls don't need to be refactored

Reply:
agreed